auto_py_to_exe/web/js/i18n.js

```diff
@@ -221,6 +240,7 @@ const translationMap = {
                 sp: 'Consola de Windows (Ocultar consola)',
                 tr: 'Pencere Tabanlı (Konsolu gizleyin)',
                 th: 'หน้าต่าง (ซ่อนคอนโซล)',
+                fr: 'Basé sur windows (la console n''est pas visible')',

```

Comment:
Looks like the last quote has not been escaped correctly. Have you run this yourself?

Reply:
Damn, I added an apostrophe too many. Yes, I translated it myself